### ISSUE TITLE
Add some sort of permission check to DELETE /api/account

### ISSUE THREAD

**Issue by Alekaei** [eejitcms/api#5]
Account deletion is currently unprotected and anyone can delete accounts, add some sort of permission check and/or password to verify account should be deleted. Should discuss later.

**Comment by ericm** [comment]
So
- [ ] Check if user is admin or username matches account being deleted?

**Comment by Alekaei** [comment]
yeah something along those lines, we havent discussed anything on permissions/roles yet so its something we will need to keep in mind
